third_party/terraform/utils/error_retry_predicates.go

```diff
@@ -205,6 +205,17 @@ func serviceUsageServiceBeingActivated(err error) (bool, string) {
 	return false, ""
 }
 
+// Retry if Bigquery operation returns a 403 with a specific message for
+// concurrent operations (which are implemented in terms of 'edit quota').
+func isBigqueryIAMQuotaError(err error) (bool, string) {
+	if gerr, ok := err.(*googleapi.Error); ok {
+		if gerr.Code == 403 && strings.Contains(strings.ToLower(gerr.Body), "Exceeded rate limits") {
```

Comment:
This will never return true because the lowercase error body will never have an E in it.

Also, this error message seems different than the one in the GH issue- intentional?

Reply:
Yep, missed that!  The full error is 

```
Error: Error creating DatasetAccess: googleapi: Error 403: Exceeded rate limits: too many dataset metadata update operations for this dataset. For more information, see https://cloud.google.com/bigquery/troubleshooting-errors, rateLimitExceeded
```

I'll lowercase the string there.